Integrations/Gmail/Gmail.py

```diff
@@ -187,37 +268,57 @@ def get_email_context(email_data, mailbox):
         # only for incident
         'Cc': headers.get('cc', []),
         'Bcc': headers.get('bcc', []),
-        'Date': headers.get('date', ''),
+        'Date': unicode(base_time),
```

Comment:
Why are you using unicode? In case base_time == None you will have `None` string.